static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -22,60 +20,67 @@ function DomMutations({replay}: Props) {
 
   const startTimestampMs = replay.getReplay().started_at.getTime();
 
+  if (isLoading) {
+    return null;
+  }
+
   return (
-    <Fragment>
-      <StyledPanelTable
-        isEmpty={actions.length === 0}
-        emptyMessage={t('No DOM actions found.')}
-        isLoading={isLoading}
-        headers={[t('Action'), t('Selector'), t('HTML'), t('Timestamp')]}
-      >
-        {actions.map((mutation, i) => (
-          <Fragment key={i}>
-            <TitleContainer>
-              <IconWrapper color={mutation.crumb.color}>
-                <BreadcrumbIcon type={mutation.crumb.type} />
-              </IconWrapper>
-              <Title>{getDetails(mutation.crumb).title}</Title>
-            </TitleContainer>
-
-            <Column>
-              <Truncate
-                maxLength={30}
-                leftTrim={(mutation.crumb.message || '').includes('>')}
-                value={mutation.crumb.message || ''}
-              />
-            </Column>
-
-            <Column>
-              <CodeContainer>
-                <HTMLCode code={mutation.html} />
-              </CodeContainer>
-            </Column>
-
-            <Column>
-              <PlayerRelativeTime
-                relativeTimeMs={startTimestampMs}
-                timestamp={mutation.crumb.timestamp}
-              />
-              {}
-            </Column>
-          </Fragment>
-        ))}
-      </StyledPanelTable>
-    </Fragment>
+    <MutationList>
+      {actions.map((mutation, i) => (
+        <MutationListItem key={i}>
+          <StepConnector />
+          <MutationItemContainer>
+            <div>
+              <MutationMetadata>
+                <IconWrapper color={mutation.crumb.color}>
+                  <BreadcrumbIcon type={mutation.crumb.type} />
+                </IconWrapper>
+                <PlayerRelativeTime
+                  relativeTimeMs={startTimestampMs}
+                  timestamp={mutation.crumb.timestamp}
+                />
+              </MutationMetadata>
+              <MutationDetails>
+                <TitleContainer>
+                  <Title>{getDetails(mutation.crumb).title}</Title>
+                </TitleContainer>
+                <Truncate
+                  maxLength={30}
+                  leftTrim={(mutation.crumb.message || '').includes('>')}
+                  value={mutation.crumb.message || ''}
+                />
+              </MutationDetails>
+            </div>
+            <CodeContainer>
+              <HTMLCode code={mutation.html} />
+            </CodeContainer>
+          </MutationItemContainer>
+        </MutationListItem>
+      ))}
+    </MutationList>
   );
 }
 
-const StyledPanelTable = styled(PanelTable)`
-  grid-template-columns: max-content max-content 1fr max-content;
-  font-size: ${p => p.theme.fontSizeSmall};
+const MutationList = styled('ul')`
+  list-style: none;
+  position: relative;
```

Comment:
This should be a scrollable area

and i like the borders we have on the network table, for example, i think it'd be nice and consistent if we had that same border on this component (and eventually all of them within the focus area)

Reply:
it's gonna need to be it's own scrollable area because we're going to need to follow up and do some perf improvements. On replays with lots of clicks,.... it's a bit slow to render all the code blocks, so many are below the fold.

Reply:
@ryan953 is this what you had in mind?

![Screen Shot 2022-08-02 at 10 32 12 AM](https://user-images.githubusercontent.com/3721977/182400209-66504983-b923-480d-9e36-21d739817ad3.png)

Reply:
ya, i think that looks good! keeps everything together

Reply:
lets set `height: 100%` instead of `max-height` tho